### PR TITLE
Rules Page: added images, text and button with link

### PR DESCRIPTION
**Pull request by athar05**
This PR consists: 

- Image 
- Instructions 
- Button with link 

This page is mobile responsive

Live preview: https://famous-empanada-1e1281.netlify.app/

### CODE REVIEW

File: src/components/Rules-Text/RulesText.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React from "react";
+
+function RulesText() {
+  return (
+    <>
+      <div className="quiz-banner">
+        <img
+          src="/assets/images/minion-all-the-best.jpg"
+          alt=""
```
Comment (bhavyaj12) [inline src/components/Rules-Text/RulesText.js]: add an alt if possible

File: src/pages/Rules.js [hunk 2]
```diff
@@ -1,7 +1,28 @@
...
-  return <div>This is rules page</div>;
+  return (
+    <div className="container">
+      <div className="nav">
+        <Navbar />
+      </div>
+      <div className="body-container-rules">
+        <RulesText />
+        <Link to="/quiz">
+          <Button variant="contained" style={{ marginTop: "2rem" }}>
```
Comment (bhavyaj12) [inline src/pages/Rules.js:16]: try to avoid inline style, and do it using classname

**Review by bhavyaj12 — COMMENTED** [review]
I think font size of the site is too big, can be reduced. Rest good work

File: src/components/Rules-Text/RulesText.js (hunk 1 above)
Comment (athar05) [reply src/components/Rules-Text/RulesText.js]: Done.

**Pull request merged**